_ml-commons-plugin/api/mcp-server-apis/sse-message.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: SSE Message 
+parent: Agent APIs
+grand_parent: ML Commons APIs
+nav_order: 50
+---
+
+# SSE Message 
+**Introduced 3.0**
+{: .label .label-purple }
+
+This is the standard message interaction endpoint of MCP protocol, you can use this API to send messages to MCP server in OpenSearch. Usually you don't need to explicitly interact with this API if you're using standard MCP client. You can checkout the example client here: https://github.com/zane-neo/opensearch-mcpserver-test-example
+
+## Endpoints
+
+```json
+POST /_plugins/_ml/mcp/sse/message
+```
+
+#### Example request
+
+```json
+POST /_plugins/_ml/mcp/sse/message
+{
+    "jsonrpc": "2.0",
+    "id": "110",
+    "method": "tools/call",
+    "params": {
+        "name": "ListIndexTool1",
+        "arguments": {
+            "input": {
+                "index": ["test"]
```

Comment:
For the ListIndexTool, the parameter that specifies index names is `indices`.  Is this one different?

Reply:
good catch, will change this.

Reply:
I changed to "indices". Could you verify that this example request works?

Reply:
It works, there's an extra curly braces and I removed it.